components/tailscale/main.tf

```diff
@@ -0,0 +1,25 @@
+module "tailscale_subnet_router" {
+  source  = "git::https://github.com/masterpointio/terraform-aws-tailscale.git?ref=tags/0.1.0"
```

Comment:
side note: @Gowiem we should publish to Terraform Registry so we can reference this as "masterpointio/tailscale/aws"

But also we should be more specific about how that repo is named, instead of "tailscale" call it "tailscale-subnet-router"

Or if it is going to be a collection of tailscale resources, then make `./modules/subnet-router`